aws-ecs/README.md.jinja

```diff
@@ -0,0 +1,176 @@
+
+Based on the type of container to monitor, there are two ways to deploy the SignalFx Smart Agent to auto-discover services and collect additional detailed metrics.
+
+#### Option 1: Monitor Amazon ECS containers
+
+##### Step 1: Create the task definition
+
+To deploy the agent on Amazon ECS, create the agent task definition from the web admin console:
+
+1. Select **Task Definitions**.
+2. Click **Create new Task Definition**.
+3. Select **EC2** and click **Next step**. The agent only supports EC2 mode and not Fargate at this time.
+4. Scroll to the bottom of the page and click **Configure via JSON**.
+5. Paste in the contents of the signalfx-agent-task.json.
+6. Click **Save.**
+7. Click the **signalfx-agent** container definition under **Container Definitions** and find the section on environment variables.
+8. Change the value of the `ACCESS_TOKEN`, `INGEST_URL`, `API_URL`, and `TRACE_ENDPOINT_URL` environment variables to the access token for your SignalFx organization and endpoints you want to send metrics to.
+9. Click **Update** and then click **Create** at the bottom of the task definition input form.
+
+You can also do this using the AWS CLI tool by issuing the following command:
+
+```
+aws ecs register-task-definition --cli-input-json file:///path/to/signalfx-agent-task.json
+```
+##### Step 2: Launch the agent
+
+**Note:** The agent is designed to be run as a daemon service in an Amazon ECS cluster.
+
+To create an agent service from the Amazon ECS web admin console:
+
+1. Go to your cluster in the web admin console.
+2. Click **Services**.
+3. Click **Create**.
+4. Make the following selections:
+  a. **Launch Type:** EC2
+  b. **Task Definition (Family):** signalfx-agent
+  c. **Task Definition (Revision):** 1 (or whatever the latest is in your case)
+  d. **Service Name:** signalfx-agent
+  e. **Service type:** DAEMON
+5. Choose the default setting for the remaining options.
+6. Click **Next step**.
+7. Choose the default setting for the remaining options.
+8. Click **Next step**.
+9. Choose the default setting for the remaining options.
+10. Click **Next step**.
+11. Click **Create Service**. The agent is deployed onto each node in the Amazon ECS cluster, and the infrastructure and docker metrics start flowing soon.
+
+##### Step 3: Configuration
+
+The main technique for configuring the agent is to have a configuration file downloaded from the network using curl in the agent container's initialization script. By default, the agent pulls from the <a href="https://github.com/signalfx/signalfx-agent/blob/master/deployments/ecs/agent.yaml" target="_blank">configuration file</a> in the GitHub repository that provides a basic configuration that might suffice for basic monitoring cases. If you wish to provide a more complex configuration file, you can set the `CONFIG_URL` environment variables in the agent task definition to the URL of the configuration file. This location must be accessible from the Amazon ECS cluster.
```

Comment:
This para includes "if you wish..." but Splunk usage dictionary says to scrub "wish" and use "want" instead.